tmt/steps/execute/scripts/tmt-report-result

```diff
@@ -10,14 +10,14 @@ die() { echo "$*" >&2; exit 2; }  # complain to STDERR and exit with error
 needs_arg() { if [ -z "$OPTARG" ]; then die "No arg for --$OPT option"; fi; }
 
 write_report_file () {
-    echo "  - name: ${TESTNAME}" >> "$REPORT_RESULT_OUTPUTFILE"
-    echo "    result: ${TESTRESULT}" >> "$REPORT_RESULT_OUTPUTFILE"
+    echo "  - name: \"${TESTNAME}\"" >> "$REPORT_RESULT_OUTPUTFILE"
```

Comment:
And now we have to hope that user does not include a `"` character in the test name :grin: 

    tmt-report-result "first:\" test" pass

Which then results into:

    Invalid yaml syntax: while parsing a block mapping
      in "<unicode string>", line 1, column 5
    did not find expected key
      in "<unicode string>", line 1, column 21

Shall we escape quotes as well?

Reply:
> Shall we escape quotes as well?

I think yes, we should escape them. Please, take a look again, the double quotes should be handled correctly now.

I also added a release note.

Reply:
Hm, and if we want to be completely precise, we should probably escape double quotes in the log file name (`$outputFile`) as well, because on Unix filesystems, quotes can appear in filenames :man_facepalming: .

There is also a `data-path`, but this seems to be always constructed correctly from TMT variables:

```
echo "    data-path: \"${TMT_TEST_DATA/$TMT_TEST_EXE}\"" >> "$REPORT_RESULT_OUTPUTFILE"
```

What do you think?

Reply:
I'd say, for now the fix is sufficient. Let's get rid of the regression and improve things in the future if needed.